docs/build.gradle

```diff
@@ -1051,7 +1051,7 @@ buildRestTests.setups['calendar_outages_addevent'] = buildRestTests.setups['cale
        ml.post_calendar_events:
          calendar_id: "planned-outages"
          body:  >
-           { "events" : [
+           { "eventBuffer" : [
```

Comment:
probably unintentional

Reply:
Yes, stupid IntelliJ. There were actually a number of other instances as well.